### ISSUE TITLE
Commit message in past builds too short

### ISSUE THREAD

**Issue by oliversalzburg** [Strider-CD/strider#703]
I just pulled the latest version of Strider from git and the commit messages in the Past Builds list seems a bit short:

![](http://i.imgur.com/WzCsFqn.png)

This is in Chrome Stable

**Comment by knownasilya** [comment]
How big is your resolution? At what point are they too short (px)?

**Comment by oliversalzburg** [comment]
I'm on 2560x1440. This is in a maximized window. Here is a screenshot of the complete panel:

![](http://i.imgur.com/xOErrkw.png)

`#build-page #list-of-builds .truncate` is set to `max-width: 110px;`

**Comment by oliversalzburg** [comment]
@knownasilya I also noticed that the past builds list no longer indicates if a particular build failed. Was that an intentional change?

**Comment by knownasilya** [comment]
Nope, it should. There are several bugs around sockets, which have yet to be tracked down. Although I haven't noticed any issues with failed builds not showing up in the "past builds" section.

**Comment by oliversalzburg** [comment]
Well, they *do* show up. It's just that you don't have an indicator that that particular build failed, until you select it and you get the red background.

**Comment by knownasilya** [comment]
Yes, that's a bug, just checked and the icon class isn't coming through.

**Comment by oliversalzburg** [comment]
@knownasilya Let me know if you want me to file a separate issue for that.

**Comment by knownasilya** [comment]
@oliversalzburg pushed up a fix for the icons.
